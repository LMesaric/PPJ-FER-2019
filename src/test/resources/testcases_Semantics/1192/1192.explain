semantic: integer constant to large
note: '100000000000' on line 1
