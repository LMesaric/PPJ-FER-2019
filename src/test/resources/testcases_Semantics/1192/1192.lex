KR_INT 1 int
IDN 1 b
OP_PRIDRUZI 1 =
MINUS 1 -
BROJ 1 100000000000
TOCKAZAREZ 1 ;
KR_INT 3 int
IDN 3 main
L_ZAGRADA 3 (
KR_VOID 3 void
D_ZAGRADA 3 )
L_VIT_ZAGRADA 3 {
KR_RETURN 4 return
BROJ 4 0
TOCKAZAREZ 4 ;
D_VIT_ZAGRADA 5 }
