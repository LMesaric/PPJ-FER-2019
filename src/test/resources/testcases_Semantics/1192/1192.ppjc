int b = -100000000000;

int main(void) {
  return 0;
}
