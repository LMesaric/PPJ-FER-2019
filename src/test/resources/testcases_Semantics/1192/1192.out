<primarni_izraz> ::= BROJ(1,100000000000)
